runtime/near-vm-runner/src/tests/cache.rs

```diff
@@ -104,39 +104,39 @@ fn make_cached_contract_call_vm(
 }
 
 #[test]
-fn test_wasmer2_artifact_output_stability() {
+fn test_near_vm_artifact_output_stability() {
```

Comment:
This test may want to be duplicated rather than updated, in case we need to change wasmer2 in near future before nearvm is fully deployed. Lets leave it up to the limited replayability to remove this sort of duplication.